static/css/frame.scss

```diff
@@ -47,6 +47,61 @@
   }
 }
 
+.promo-panel {
+  .edit {
+    right: 50px;
+    top: 75px;
+  }
+  .promo-box {
+    border-radius: 3px;
+    box-shadow: 0 0 0 1px rgba(0, 0, 0, 0.3), 0 0 20px rgba(0, 0, 0, 0.2);
+    position: absolute;
```

Comment:
The promo card is intended to be overlaid on top of the page, not participate in the page's layout, so it would make more sense to position the outermost element, the .promo-panel itself.